pkg/hubble/parser/options/options.go

```diff
@@ -72,6 +73,16 @@ func WithNetworkPolicyCorrelation(logger *slog.Logger, enabled bool) Option {
 	}
 }
 
+// WithSkipUnknownCGroupIDs configures whether Hubble will skip events with unknown CGroup IDs.
+func WithSkipUnknownCGroupIDs(logger *slog.Logger, enabled bool) Option {
+	return func(opt *Options) {
+		opt.SkipUnknownCGroupIDs = enabled
+		logger.Info("configured Hubble to skip events with unknown CGroup IDs",
+			logfields.Options, opt.SkipUnknownCGroupIDs,
```

Comment:
not blocking, but I feel theses log are creeping up all the Hubble parser options recently. Do we already log the Hubble cell options? IMO it should be enough to distinguish between a user configuration error and and bug, and as a bonus we wouldn't need to pass a logger to all the `With*` funcs.

Reply:
https://github.com/cilium/cilium/pull/39639